DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -924,12 +925,12 @@ public IActionResult SetSupervisorRole(SetSupervisorRoleViewModel model)
 
         [Route("/LearningPortal/SelfAssessment/{selfAssessmentId:int}/Supervisors/Add/Summary")]
         [ResponseCache(CacheProfileName = "Never")]
-        [TypeFilter(

```

Comment:
@kevwhitt-hee - I have removed the[TypeFilter] and included the session check inside the action, as in the ticket it is mentioned to redirect to the 410 page.